### ISSUE TITLE
Sync batch norm

### ISSUE THREAD

**Issue by fsx950223** [google/automl#384]
https://github.com/google/automl/blob/c84956a6c06b1b11c04ffe39949548e21ab1d68b/efficientdet/utils.py#L223

Is it should be ```tf.keras.layers.experimental.SyncBatchNormalization```?

**Comment by mingxingtan** [comment]
Good point, I have never used tf.keras.layers.experimental.SyncBatchNormalization before. Seems like it needs to be coupled with distributed_strategy?

**Comment by fsx950223** [comment]
If this layer is used when using tf.distribute strategy to train models across devices/workers, there will be an allreduce call to aggregate batch statistics across all replicas at every training step. Without tf.distribute strategy, this layer behaves as a regular tf.keras.layers.BatchNormalization layer.
